src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java

```diff
@@ -1948,7 +1948,7 @@ private static LambdaForm makeLoopForm(MethodType basicType, BasicType[] localVa
         }
         @Override
         public String toString() {
-            StringBuffer sb = new StringBuffer("LoopClauses -- ");
+            StringBuilder sb = new StringBuilder("LoopClauses -- ");
             for (int i = 0; i < 4; ++i) {
                 if (i > 0) {
                     sb.append("       ");
```

Comment:
Consider using `String.repeat` here and on L1960 for clarity.

Reply:
I don't think it can be used here.